### ISSUE TITLE
Report Doorbell Properties Similar to Cameras

### ISSUE THREAD

**Issue by sissiwup** [fronzbot/blinkpy#638]
{'name': 'CamB0', 'camera_id': '1234', 'serial': '12345', 'temperature': None, 'temperature_c': None, 'temperature_calibrated': None, 'battery': 'ok', 'battery_voltage': None, 'thumbnail': '*', 'video': None, 'motion_enabled': True, 'motion_detected': False, 'wifi_strength': None, 'network_id': 123, 'sync_module': 'syncName', 'last_record': None, 'type': 'lotus'}

Blink Doorbell did not deliver voltage, temp and wifi_strength. A Doorbell can you recognize from type "lotus".
Can you integrate Doorbells as well as Cameras?

**Comment by fronzbot** [comment]
API endpoint has to be uncovered to get that information. It would be great to include it once (if) it's found!

**Comment by not10robots** [comment]
I was able to identify and verify the doorbell configuration endpoint used by the Blink app.

For Blink doorbells, the app requests:

```text
GET /api/v1/accounts/{account_id}/networks/{network_id}/doorbells/{doorbell_id}/config
```

The response includes the missing sensor fields:

- `temp`
- `wifi_rssi`
- `lfr_rssi`
- `battery_state`

Tested successfully with both:

- Video Doorbell 2nd Gen (`tulip`)
- Battery Doorbell 2K+ (`orchid`)

Sanitized example responses:

```json
{
  "temp": 86,
  "wifi_rssi": -54
}
```

```json
{
  "temp": 90,
  "wifi_rssi": -41
}
```

The current `BlinkDoorbell.get_sensor_info()` method is empty, so these values remain unavailable in `blinkpy` and Home Assistant. Calling the endpoint above and mapping `temp` to the temperature value and `wifi_rssi` to Wi-Fi strength exposes them correctly.

No credentials, tokens, account IDs, or device IDs are included here.
